src/update_handler.c

```diff
@@ -785,6 +785,7 @@ static gboolean run_slot_hook(const gchar *hook_name, const gchar *hook_cmd, Rau
 	}
 	if (image) {
 		g_subprocess_launcher_setenv(launcher, "RAUC_IMAGE_NAME", image->filename, TRUE);
+		g_subprocess_launcher_setenv(launcher, "RAUC_IMAGE_SIZE", g_strdup_printf("%ld", image->checksum.size), TRUE);
```

Comment:
`g_strdup_printf` was the first result I found when I googled `glib int to string`. If you prefere another way, please let me know

Reply:
`g_strdup_printf()` is okay, but please ensure we do not leak the string (thus assigned it to a `g_autofree gchar*`) and use the define `G_GOFFSET_FORMAT` instead of "`ld`".